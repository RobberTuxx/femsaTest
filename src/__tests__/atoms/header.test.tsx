import React from 'react';
import renderer from 'react-test-renderer';
import Header from "../../components/atoms/header";
it('render header', ()=> {
    const component = renderer.create(<Header text={"Hello world!"}/>).toJSON()
    expect(component).toMatchSnapshot()
})
it('has 0 child', ()=> {
    const component = renderer.create(<Header text={"Hello world!"}/>).toJSON()
     expect(component.children.length).toBe(1)
})
