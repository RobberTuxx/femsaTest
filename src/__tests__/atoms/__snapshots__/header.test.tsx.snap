// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render header 1`] = `
<Text
  style={
    {
      "color": "#9B9898",
      "fontFamily": "Avenir",
      "fontSize": 22,
      "fontWeight": "900",
      "marginBottom": 31.5,
      "marginTop": 28,
    }
  }
>
  Hello world!
</Text>
`;
